crates/ln-dlc-node/src/ln/dlc_channel_details.rs

```diff
@@ -52,16 +54,44 @@ pub enum ChannelState {
 
 impl From<Channel> for DlcChannelDetails {
     fn from(channel: Channel) -> Self {
-        let (update_idx, state, fee_rate_per_vb, funding_txid, funding_tx_vout) =
+        let (update_idx, state, fee_rate_per_vb, funding_txid, funding_tx_vout, closing_txid) =
             match channel.clone() {
+                Channel::Signed(SignedChannel {
+                    update_idx,
+                    fee_rate_per_vb,
+                    fund_tx,
+                    fund_output_index,
+                    state: dlc_manager::channel::signed_channel::SignedChannelState::CollaborativeCloseOffered{close_tx, ..},
+                    ..
+                }) => (
+                    Some(update_idx), Some(SignedChannelState::CollaborativeCloseOffered), Some(fee_rate_per_vb), Some(fund_tx.txid().to_hex()), Some(fund_output_index), Some(close_tx.txid().to_hex())),
```

Comment:
🔧 Doesn't `to_string` work too? There is no `ToHex` in `bitcoin:0.30`, so I would avoid using it.